modules/gitops-inbuilt-permissions-for-cluster-config.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assembly:
+//
+// * configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-inbuilt-permissions-for-cluster-config_{context}"]
+= In-built permissions for cluster configuration
+
+By default, the Argo CD instance has permissions to manage specific cluster-scoped resources such as platform Operators, optional OLM Operators and user management. 
+
+[NOTE]
+====
+Argo CD does not have cluster-admin permissions.
+====
+
+The following table lists the permissions for the Argo CD instance:
```

Comment:
```suggestion
.Permissions for the Argo CD instance
```

Reply:
IBM SG:
"Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes". Instead, focus on
the purpose or value of the content, and include words that users are likely to search for. For example, in a short
description in an online help topic, provide users with enough information to determine whether the topic meets their
needs. You can use self-referential text in learning modules and tutorials."